apps/web-giddh/src/app/reports/components/purchase-register-component/purchase.register.component.ts

```diff
@@ -110,8 +113,12 @@ export class PurchaseRegisterComponent implements OnInit, OnDestroy {
         private companyService: CompanyService,
         private _toaster: ToasterService,
         private settingsBranchAction: SettingsBranchActions,
-        private generalService: GeneralService) {
-
+        private generalService: GeneralService, private breakPointObservar: BreakpointObserver,) {
```

Comment:
Always add new dependencies in constructor on a new line
```suggestion
        private generalService: GeneralService, 
        private breakPointObservar: BreakpointObserver) {
```